### PR TITLE
Add YML files to build and publish WindowsAppSDKTelemetry library and headers

### PR DESCRIPTION
**Pull request by ZenBird**
Add a YML script to publish the "WindowsAppSDKTelemetry.h" into WindowsAppSDK core package.

### CODE REVIEW

File: build/publish-WindowsAppSDKTelemetry.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+parameters:
```
Comment (kythant) [inline build/publish-WindowsAppSDKTelemetry.yml:1]: nit: Could you put this file in build/AzurePipelineTemplates and call it build/publish-WindowsAppSDKTelemetry-Steps.yml

**Review by kythant — APPROVED** [review]

**Comment by ZenBird** [comment]
completing PR

**Pull request merged**